pages/[id].jsx

```diff
@@ -0,0 +1,12 @@
+import { useRouter } from 'next/router';
+
+export default function DetailPage() {
+  const router = useRouter();
+  const { id } = router.query;
+
+  return (
+    <h1>
+      {`Hello ${id}!`}
+    </h1>
```

Comment:
For the sake of making it easier to add things later and being consistent with the other PR, let's add in a little more html structure.

```suggestion
    <div>
      <Head>
        <title>{`${id} | Sparkbox Team Availability`}</title>
        <meta name="description" content={`View details about ${id}, including their projects, skills, and interests.`} />
      </Head>

      <main>
        <h1>
          {`Hello ${id}!`}
        </h1>
      </main>
    </div>
```

Reply:
I agree, these changes make sense.